cms/templates/studio_xblock_wrapper.html

```diff
@@ -47,7 +47,11 @@
 % endif
 
 <header class="xblock-header xblock-header-${xblock.category}">
```

Comment:
header elements should be children of articles.  Can we move this inside the article below?